### PR TITLE
logging event data

### PR DESCRIPTION
**Pull request by pankajpatel**
logging:
- event data in the plugin panel
- complete event in the console

### CODE REVIEW

**Comment by pankajpatel** [comment]
event-data

**Comment by pankajpatel** [comment]
fixes #30

**Comment by nogizhopaboroda** [comment]
Why not to use inspector component?

File: addons/events/index.js [hunk 1]
```diff
@@ -89,13 +122,17 @@ class EventsPreview extends HTMLElement {
 
     this.channel.emit(EVENTS_LIST_UPDATE, events);
 
-    events.forEach(eventName => this.addEventListener(eventName, ({ type }) => {
-      this.channel.emit(EVENT_FIRED, type);
+    events.forEach(eventName => this.addEventListener(eventName, e => {
+      console.log(e);
+      this.channel.emit(EVENT_FIRED, {
```
Comment (nogizhopaboroda) [inline addons/events/index.js]: seems like you can pass the whole event here and pick needed properties in manager

File: addons/events/index.js [hunk 2]
```diff
@@ -49,21 +75,28 @@ class EventsManager extends HTMLElement {
...
+      this.innerHTML = styles + eventsCode + list;
+      this.list = this.querySelector('#events-data');
     });
 
     this.channel.on(EVENT_FIRED, event => {
-      this.addClassFor(document.querySelector(`#${event}`), "fired", ANIMATION_DURATION);
+      this.addClassFor(document.querySelector(`#${event.type}`), "fired", ANIMATION_DURATION);
+      const li = document.createElement('li');
+      const info = [];
+      info.push({ label: +new Date, value: event.type });
```
Comment (nogizhopaboroda) [inline addons/events/index.js]: Shouldn't it be more human-readable format than timestamp?

**Comment by pankajpatel** [comment]
`inspector-component` has some issues that need to be fixed first before being added here

**Review by nvignola — APPROVED** [review]

**Comment by pankajpatel** [comment]
Updated the code now. Looks like following:
events-logging

for following usage:
```js
storiesOf('My Component')
  .addDecorator(withEvents(['click', { type: 'change', extract: ['value', 'target.value'] }]))
  .add('default', () => `
    <my-component></my-component>
  `)
```

**Review by nogizhopaboroda — APPROVED** [review]
awesome

**Pull request merged**